site/styles/_pages.scss

```diff
@@ -76,6 +98,17 @@
   }
 }
 
+.curated-page,
+.collection-page {
```

Comment:
Is this block made redundant by the block on line 72?

Reply:
Yes. Removed.